src/platforms/crc.ts

```diff
@@ -0,0 +1,79 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+// tslint:disable:object-curly-spacing
+
+import { Command } from '@oclif/command'
+import * as commandExists from 'command-exists'
+import * as execa from 'execa'
+import * as Listr from 'listr'
+
+/**
+ * Helper for Code Ready Container
+ */
+export class CRCHelper {
+  startTasks(flags: any, command: Command): Listr {
+    return new Listr([
+      {
+        title: 'Verify if oc is installed',
+        task: (_ctx: any, task: any) => {
+          if (!commandExists.sync('oc')) {
+            command.error('E_REQUISITE_NOT_FOUND')
+          } else {
+            task.title = `${task.title}...done.`
+          }
+        }
+      },
+      { title: 'Verify if crc is installed',
+        task: (_ctx: any, task: any) => {
+          if (!commandExists.sync('crc')) {
+            command.error('E_REQUISITE_NOT_FOUND', { code: 'E_REQUISITE_NOT_FOUND' })
+          } else {
+            task.title = `${task.title}...done.`
+          }
+        }
+      },
+      { title: 'Verify if CodeReady Containers is running',
+        task: async (_ctx: any, task: any) => {
+          const crcIsRunning = await this.isCRCRunning()
+          if (!crcIsRunning) {
+            command.error('E_PLATFORM_NOT_READY')
+          } else {
+            task.title = `${task.title}...done.`
+          }
+        }
+      },
+      { title: 'Retrieving CodeReady Containers IP and domain for routes URLs',
+        enabled: () => flags.domain !== undefined,
+        task: async (_ctx: any, task: any) => {
+          const ip = await this.getCRCIP()
+          flags.domain = ip + '.nip.io'
+          task.title = `${task.title}...${flags.domain}.`
+        }
+      },
+    ], {renderer: flags['listr-renderer'] as any})
+  }
+
+  async isCRCRunning(): Promise<boolean> {
+    const { code, stdout } = await execa('crc', ['status'], { timeout: 60000, reject: false })
```

Comment:
it looks like i don't have the status parameter with crc
```bash
$ crc version
version: 0.88.0-alpha-4.1.3+783056a

 $ crc status
Error: unknown command "status" for "crc"
Run 'crc --help' for usage.
FATA[0000] unknown command "status" for "crc"           


```

Reply:
I compiled CRC from master so probably you'll have it in the upcoming release